patterns/footer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Title: Default footer
+ * Slug: block-canvas/footer
```

Comment:
`block-canvas` => `course`

Reply:
Removed all the instances of block-canvas in the new PR: https://github.com/Automattic/sensei-theme/pull/26/files#diff-1acd65465d61694f280f8f68d19e74df0e04128facf93e825c1ccdce9a89d1d0R4